rocAL/rocAL/include/meta_data/meta_node_crop.h

```diff
@@ -32,7 +32,7 @@ class CropMetaNode:public MetaNode
 {
     public:
         CropMetaNode() {};
-        void update_parameters(MetaDataBatch* input_meta_data)override;
+        void update_parameters(MetaDataBatch* input_meta_data, bool segmentation)override;
```

Comment:
can we use const bool segmentation as this value is not going to be modified

Reply:
Resolved in the latest commit